linkcovery/cli/config.py

```
@@ -85,3 +132,61 @@ def reset() -> None:
+
+    Examples:
+        linkcovery config edit
+
+    """
+    config_manager = get_config_manager()
+
+    try:
+        config_file = str(config_manager._config_file)
+
+        if platform.system() == "Windows":
+            os.system(f'start "" "{config_file}"')
```

Ruff (0.15.0)

[error] Starting a process with a shell, possible injection detected

(S605)